### ISSUE TITLE
Does not work with Prusa FW 3.4.0

### ISSUE THREAD

**Issue by EddyMI3d** [amsbr/OctoPrint-EEPROM-Marlin#35]
Hi!
Version 1.2.1 (recent one?) does not work with Prusa-Firmware Version 3.4.0

![grafik](https://user-images.githubusercontent.com/8533327/45883059-47b8fe80-bdb0-11e8-8074-e5c0ad3df7e4.png)

Running OctoPrint V 1.3.9 on OctoPi 0.15.1

![grafik](https://user-images.githubusercontent.com/8533327/45883174-9b2b4c80-bdb0-11e8-8296-572afbcef693.png)

**Comment by draeath** [comment]
Er, this is Marlin based firmware?

**Comment by EddyMI3d** [comment]
Yes, it is. It worked with previous versions. But not since 3.4.0
